ResearchKit/ActiveTasks/ORKTowerOfHanoiTowerView.m

```diff
@@ -66,7 +66,8 @@ - (instancetype)initWithFrame:(CGRect)frame maximumNumberOfDisks:(NSUInteger)max
 
 - (void)updateConstraints {
     if (_currentConstraints != nil) {
-        [NSLayoutConstraint deactivateConstraints:_currentConstraints];
+        [self removeConstraints:_currentConstraints];
```

Comment:
Weird, in my super-simple aforementioned Xcode example it did crash when using `deactivateConstraints:`, but it did not when using the loop. I guess the bug is more subtle that I thought.

Again, let's hope they fix it on the *iOS 9* codebase. ;-)